components/telemetry-operator/README.md

```diff
@@ -127,6 +119,11 @@ The `global`, `tracing`, and `logging` fields are optional.
    ```bash
    kubectl rollout restart deployment -n kyma-system telemetry-operator
    ```
-   
+
 **Caveats**
 If you change the pipeline CR when the reconciliation is paused, these changes will not be applied immediately but in a periodic reconciliation cycle of one hour. To reconcile earlier, restart the telemetry operator.
```

Comment:
My suggestion would be to capitalize "Telemetry" because I believe it refers to the Kyma component, and write the operator like `Telemetry operator`. 
However, I might be wrong and other conventions might have been agreed since I last looked at that. Anyway, we should have a consistent way to write those module managers. 

```suggestion
If you change the pipeline CR when the reconciliation is paused, these changes will not be applied immediately but in a periodic reconciliation cycle of one hour. To reconcile earlier, restart the Telemetry operator.
```